torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1881,14 +1885,16 @@ def remove_redundant_dynamic_results(self):
     def prettify_results(
         self,
         original_signature: inspect.Signature,
+        dynamic_shapes=None,
```

Comment:
How does this help with determinism?

Reply:
Sorry this is unclean, this is left over from trying to order results with the position in the shapes spec. No longer doing that.